src/index.js

```diff
@@ -109,119 +110,56 @@ async function retrievePDF(order) {
 // even if this in another domain here, but it works as an example
 async function authenticate(username, password) {
   const xsrfToken = await getXsrfToken()
-  const loginResponse = await request({
+  const options = {
+    xsrfToken,
+    username,
+    password,
+    url: `${baseURL.origin}/api/reef/login`,
+    validate: ({ body: { status } }) => status
+  }
+
+  const isSigned = await signin(options)
+  if (!isSigned) {
+    throw new Error('Unable to signin, check the username and/or the password')
```

Comment:
Errors should be one of https://github.com/konnectors/libs/blob/master/packages/cozy-konnector-libs/src/helpers/errors.js for automatic handling by the stack. Used log to add more details if needed.